### DISCUSSION TITLE
Deprecated dependencies

### DISCUSSION THREAD

**Discussion by jaufderheide** [sveltejs/kit#12263] [General]
On npm create of new skeleton app ( `npm create svelte@latest`) I get this response: 

npm install
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated rimraf@2.7.1: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported

Any suggestions here?

**Comment by teemingc** [comment]
Related to https://github.com/sveltejs/kit/issues/12258
